fury/utils.py

```diff
@@ -1073,3 +1073,43 @@ def get_bounds(actor):
 
     """
     return actor.GetMapper().GetInput().GetBounds()
+
+
+def clip_overflow(textblock, width):
```

Comment:
This function should stay in UI, later, we will split the UI modules into multiple files.

Reply:
This should be outside of any class definations right?

Reply:
Yes